src/components/Modals/FiatRamps/fiatRampProviders/juno-pay.ts

```diff
@@ -0,0 +1,55 @@
+import { adapters } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import { getConfig } from 'config'
+import { logger } from 'lib/logger'
+
+import { FiatRampAction, FiatRampAsset } from '../FiatRampsCommon'
+
+const moduleLogger = logger.child({
+  namespace: ['Modals', 'FiatRamps', 'fiatRampProviders', 'JunoPay'],
+})
+
+export async function getJunoPayAssets(): Promise<FiatRampAsset[]> {
+  try {
+    const junoPayData  = await axios.get(getConfig().REACT_APP_JUNOPAY_SUPPORTED_COINS)
+    
+    let junoPayToCurrencyList= junoPayData.data.data.settings.buy.to_currency
+    
+    let junoPayCurrencyList = junoPayToCurrencyList.map((item: any) => {
+      return item;
+    });
+    let allCurrencyList = junoPayData.data.data.settings.metadata 
+    
+    const junoPayAssets = allCurrencyList.filter((item: { short_name: string }) => junoPayCurrencyList.includes(item.short_name.toUpperCase()));
+    
+    const assets: FiatRampAsset[] = junoPayAssets.map((asset: { long_name: string; short_name: string; logo_url: string; }) => ({
+      assetId: adapters.junopayTickerToAssetId(asset.short_name),
```

Comment:
Not something that needs to be done in the scope of this PR but rather an overlook in the fiat ramps abstraction: If we make this a regular function body without the shorthand return syntax, and do
`const assetId = adapters.junopayTickerToAssetId(asset.short_name)`

Then we can actually surface the symbol, name, and asset image URL from our own`Asset` type.